src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -74,6 +74,10 @@ public static Phone parsePhone(String phone) throws ParseException {
     public static Address parseAddress(String address) throws ParseException {
         requireNonNull(address);
         String trimmedAddress = address.trim();
+        if (trimmedAddress.isEmpty()) {
```

Comment:
Do we need the check here? Since `Address.isValidAddress` has already been updated.